doc/source/howtos/upgrade_guide.rst

```diff
@@ -4,14 +4,16 @@
 Upgrading Percona XtraDB Cluster
 ================================
 
-This guide describes the procedure for upgrading |PXC| version 5.6
-to the latest version 5.7 without downtime (*rolling upgrade*).
-This means that replication between nodes will continue during the upgrade.
-
-Rolling upgrade from earlier versions to 5.7 is not supported.
-If you are running |PXC| version 5.5, it is recommended to shut down all nodes,
-then remove and re-create the cluster from scratch.
-Alternatively, you can perform a `rolling upgrade from PXC 5.5 to 5.6 <https://www.percona.com/doc/percona-xtradb-cluster/5.6/upgrading_guide_55_56.html>`_,
+This guide describes the procedure for upgrading |PXC| without downtime
+(*rolling upgrade*) to the latest 5.7 version. Rolling upgrade means that
+replication between nodes will continue during the upgrade.
```

Comment:
Rolling upgrade means no need to take down complete cluster during the upgrade.

Reply:
With a rolling upgrade, the cluster remains active during the upgrade.